pylmod/gradebook.py

```diff
@@ -948,3 +1024,152 @@ def spreadsheet2gradebook(
+                                u'email': u'hduck@mit.edu',
+                                u'givenName': u'Huey',
+                                u'middleName': None,
+                                u'mitId': u'920445024',
+                                u'nickName': u'Huey',
+                                u'personId': 1299059,
+                                u'sortableName': u'Duck, Huey',
+                                u'surname': u'Duck',
+                                u'year': None
+                            },
+                            {
+                                u'accountEmail': u'dduck@mit.edu',
+                                u'displayName': u'Donald Duck',
+                                u'editable': False,
+                                u'email': u'dduck@mit.edu',
+                                u'givenName': u'Donald',
+                                u'middleName': None,
+                                u'mitId': u'916144889',
+                                u'nickName': u'Donald',
+                                u'personId': 8117160,
+                                u'sortableName': u'Duck, Donald',
+                                u'surname': u'Duck',
+                                u'year': None
+                            },
+                        ],
+                        u'COURSE_TA': [
+                            {
+                                u'accountEmail': u'benfranklin@mit.edu',
+                                u'displayName': u'Benjamin Franklin',
+                                u'editable': False,
+                                u'email': u'benfranklin@mit.edu',
+                                u'givenName': u'Benjamin',
+                                u'middleName': None,
+                                u'mitId': u'921344431',
+                                u'nickName': u'Benjamin',
+                                u'personId': 10710616,
+                                u'sortableName': u'Franklin, Benjamin',
+                                u'surname': u'Franklin',
+                                u'year': None
+                            }
+                        ]
+                    },
+                    u'message': u'',
+                    u'status': 1
+                }
+
+
+        """
+        staff_data = self.get(
+            'staff/{gradebookId}'.format(
+                gradebookId=gradebook_id or self.gradebook_id
+            ),
+            params=None,
+        )
+        if simple:
+            simple_list = []
+            unraveled_list = self.unravel_staff(staff_data)
+            for member in unraveled_list.__iter__():
+                simple_list.append({
+                    'accountEmail': member['accountEmail'],
+                    'displayName': member['displayName'],
+                    'role': member['role'],
+                })
+            return simple_list
+        return staff_data
```

Comment:
shouldn't this be staff_data['data'] like the others, or does it not have that key?